scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -92,6 +92,9 @@ task CollectCounts {
     Int? preemptible_attempts
     Int? disk_space_gb
 
+    Int machine_mem = select_first([mem, 8]) * 1000
```

Comment:
As far as small optimizations go, if these tasks that are asking for `8000 MB` by default can make due with `7500 MB` then you can save 25% on GCP compute - https://cloud.google.com/compute/pricing#machinetype - `n1-standard-2` vs`n1-highmem-2`

Reply:
OK, good to know!  No particular reason these need Xmx8G, so I'll change them.